protocol/message.go

```diff
@@ -1,87 +1,152 @@
+// Defines the message formats of the CONIKS protocols
```

Comment:
Nitpicking: should be consistent with the description in doc.go "message format" (singular or plural)